packages/strapi-plugin-users-permissions/controllers/Auth.js

```diff
@@ -394,10 +394,14 @@ module.exports = {
       );
     }
 
-    const params = _.assign(ctx.request.body, {
+    let params = _.assign(ctx.request.body, {
       provider: 'local',
     });
 
+    if (!ctx.state.admin) {
```

Comment:
Can you change this to 

```js
const params = Object.assign(_.pick(ctx.request.body, ['username', 'email', 'password']), {
provider: 'local'
})
```

I don't think there is any reason this route should allow other params if it is called by and admin. There are administration routes for this instead